torchci/rockset/commons/__sql/recent_pr_workflows_query.sql

```diff
@@ -22,7 +22,7 @@ WITH recent_shas AS (
 SELECT
   w.id AS workflow_id,
   j.id,
-  j.name,
+  CONCAT(w.name, ' / ', j.name) as name,
```

Comment:
I'm checking if Dr.CI `isBrokenTrunk` and `hasSimilarFailures` still works correctly after this change.  These twos run `removeJobNameSuffix` and match job names.  Please help double check too.

Reply:
<!-- drci-comment-start -->

## :link: Helpful Links
### :test_tube: See artifacts and rendered test results at [hud.pytorch.org/pr/109584](https://hud.pytorch.org/pr/109584)
* :page_facing_up: Preview [Python docs built from this PR](https://docs-preview.pytorch.org/pytorch/pytorch/109584/index.html)
* :page_facing_up: Preview [C++ docs built from this PR](https://docs-preview.pytorch.org/pytorch/pytorch/109584/cppdocs/index.html)
* :question: Need help or want to give feedback on the CI? Visit the [bot commands wiki](https://github.com/pytorch/pytorch/wiki/Bot-commands) or our [office hours](https://github.com/pytorch/pytorch/wiki/Dev-Infra-Office-Hours)

Note: Links to docs will display an error until the docs builds have been completed.


## :x: 6 New Failures
As of commit 6fdc6213055ccadfccf0930ae9e999ba104a1530 with merge base e276d70451efdbdff599ce97e986c447063dc3fb (<sub><sub><img alt="image" width=70 src="https://img.shields.io/date/1694453884?label=&color=FFFFFF&style=flat-square"></sub></sub>):
<details open><summary><b>NEW FAILURES</b> - The following jobs have failed:</summary><p>

* [pull / linux-focal-cuda11.8-py3.10-gcc9 / test (distributed, 1, 3, linux.8xlarge.nvidia.gpu)](https://hud.pytorch.org/pr/pytorch/pytorch/109584#16942099488) ([gh](https://github.com/pytorch/pytorch/actions/runs/6239358969/job/16942099488))
* [pull / linux-focal-py3.11-clang10 / test (dynamo, 1, 2, linux.2xlarge)](https://hud.pytorch.org/pr/pytorch/pytorch/109584#16942102036) ([gh](https://github.com/pytorch/pytorch/actions/runs/6239358969/job/16942102036))
* [pull / linux-focal-py3.8-clang10 / test (dynamo, 1, 2, linux.2xlarge)](https://hud.pytorch.org/pr/pytorch/pytorch/109584#16942095433) ([gh](https://github.com/pytorch/pytorch/actions/runs/6239358969/job/16942095433))
* [pull / linux-jammy-py3.8-gcc11 / test (distributed, 1, 2, linux.2xlarge)](https://hud.pytorch.org/pr/pytorch/pytorch/109584#16942093819) ([gh](https://github.com/pytorch/pytorch/actions/runs/6239358969/job/16942093819))
* [trunk / linux-focal-rocm5.6-py3.8 / test (default, 3, 3, linux.rocm.gpu)](https://hud.pytorch.org/pr/pytorch/pytorch/109584#17121490206) ([gh](https://github.com/pytorch/pytorch/actions/runs/6305865216/job/17121490206))
* [trunk / win-vs2019-cpu-py3 / test (default, 2, 3, windows.4xlarge.nonephemeral)](https://hud.pytorch.org/pr/pytorch/pytorch/109584#17120709688) ([gh](https://github.com/pytorch/pytorch/actions/runs/6305865216/job/17120709688))
</p></details>


This comment was automatically generated by Dr. CI and updates every 15 minutes.
<!-- drci-comment-end -->

Reply:
switched to querying for workflow name in addition to job name, and constructing the full/display name in code instead